charts/kubewarden-defaults/values.yaml

```diff
@@ -3,6 +3,61 @@
 # by more than one chart and they ideally need to match during the
 # installation of the charts consuming this values.
 global:
+  # affinity:
```

Comment:
I believe that instead of having the repetition on this commented configuration, this should go to `common-values.yaml`, and be checked that way. It may be needed to edit the `make check-common-values` as needed.

Reply:
I'm fine with that, but the `make check-common-values` CI job would have failed. I would also add them into the common-values.yaml, and if the make works, it will check that they are all in sync at least.

Reply:
The CI failed for another reason. The CI is failing because the script is using a CLI flag not available in the `yq` installed in the github runner. There is a `yq` implementation in go lang and another one using python (the one with the required CLI flag). The script is considering one `yq` but the runner has another installed. Which misses the '--sort-keys' CLI flag. We need to update the script to remove the need of `--sort-keys` or install the right `yq` version. 

The CI is failing for a while now. If you check the CI history they are green. But if you read the logs, the script failed with the same problem. I've opened another issue to attack this problem. It seems out of scope of this PR. I've run the `make check-common-values` locally and it's passing. I have the python `yq` installed and it does not consider comment when comparing the values:

```
 ~/SUSE/helm-charts │ issue756 !1 ?1  make check-common-values                                                                                                                                                                ✔ │ 11:46:13 
 ~/SUSE/helm-charts │ issue756 !1 ?1                                                                                                                                                                                          ✔ │ 14:09:58 

```

Reply:
fair enough then, approving.